src/tsconfig.json

```diff
@@ -9,7 +9,7 @@
         "outDir": "../lib"
     },
     "atom": {
-        "rewriteTsconfig": false
+        "rewriteTsconfig": true
```

Comment:
I don't think we should turn this on. It makes it so devs working on tslint have to be using atom to generate the correct tsconfig.json file. Right now it's updated as part of the build - this is better because it works the same for everyone.